tests/tests.js

```diff
@@ -1,10 +1,108 @@
-const { expect } = require('chai')
-const { describe, it } = require('mocha')
+const chai = require('chai')
+const {
+  describe, it, beforeEach, afterEach,
+} = require('mocha')
 const employee = require('./employee')
 const products = require('./products')
 const pricing = require('../pricing')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+
+const { expect } = chai
+chai.use(sinonChai)
+
+
+describe('pricing', () => {
+  describe('formatPrice', () => {
+    it('returns a price with two decimal places not rounded when given a number with multiple decimal places', () => {
+      const formattedPrice = pricing.formatPrice(15.1566663)
+
+      expect(formattedPrice).to.equal(15.15)
+    })
+  })
+
+
+  describe('getEmployerContribution', () => {
+    it('returns the amount  from long term disability plan contributed by employer in dollars', () => {
+      const dollarAmount = 50
+      const price = pricing.getEmployerContribution(products.longTermDisability.employerContribution, dollarAmount)
+      expect(price).to.equal(10)
+    })
+  })
+
+
+  describe('calculateVolLifePricePerRole', () => {
+
+
+    it('returns the voluntary life price for a employee per their role', () => {
+      const role = 'ee'
+      const coverageLevel = [{ role: 'ee', coverage: 120000 }]
+
+      const price = pricing.calculateVolLifePricePerRole(role, coverageLevel, products.voluntaryLife.costs)
+      expect(price).to.equal(42)
+    })
+  })
+
+
+  it('returns the voluntary life price for a employee', () => {
+
+    const selectedOptions = {
+      familyMembersToCover: ['ee'],
+      coverageLevel: [
+        { role: 'ee', coverage: 200000 },
+      ],
+    }
+    const price = pricing.calculateVolLifePrice(products.voluntaryLife, selectedOptions)
+    expect(price).to.equal(70)
+  })
+})
+
+describe('calculateLTDprice', () => {
+  it('returns the price of an LTD plan without employer contribution factored in', () => {
+    const selectedOptions = {
+      familyMembersToCover: ['ee']
+    }
+    const price = pricing.calculateLTDPrice(products.longTermDisability, employee, selectedOptions)
+    expect(price).to.equal(32.04)
+  })
+})
+describe('calculateCommuterPrice', () => {
+  it('returns price of package based on type of commute', () => {
```

Comment:
This it should specify that this test is for train just as the other specifies it is for parking since that is the major difference in the two tests.